app/views/organization_timeline_entries/index.html.erb

```diff
@@ -12,6 +12,9 @@
       <th><%= model_class.human_attribute_name(:started_at) %></th>
       <th><%= model_class.human_attribute_name(:ended_at) %></th>
       <th>Duration</th>
+      <% if can?(:update, Task) or can?(:destroy, Task) %>
```

Comment:
This should be can?(:update, OrganizationTimelineEntry), nothing to do with Task.
@pkoenig10 For this feature I've been assuming only admins can edit this info, correct?